lib/pollingState.ts

```diff
@@ -17,15 +17,15 @@ export default class PollingState {
   /**
    * @param {msRest.RequestOptionsBase} [optionsOfInitialRequest] - Request options that were provided as a part of the initial request.
    */
-  optionsOfInitialRequest: msRest.RequestOptionsBase;
+  optionsOfInitialRequest!: msRest.RequestOptionsBase;
```

Comment:
These definite assignment assertions fix errors where TypeScript can't tell that we assigned some of these properties during/immediately after construction.